.github/renovate.json

```diff
@@ -18,6 +18,11 @@
       "matchManagers": ["github-actions"],
       "groupName": "GitHub Actions"
     },
+    {
```

Comment:
you might also want a top-level entry
```
  "baseBranches": ["main", "/^1\\..*\\.x/"],
```
as in https://github.com/janus-idp/operator/pull/317/files#diff-f82f7d36a61d22f640c25bdb8b0435bf9cb38679d99d5f460753fa668ee7cdb3R9